utils/ICanRotate.hpp

```diff
@@ -0,0 +1,22 @@
+/*
+** EPITECH PROJECT, 2024
+** ArcadeShared
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#pragma once
+
+class ICanRotate {
+
+const float UP = 90;
+const float DOWN = 270;
+const float LEFT = 180;
```

Comment:
Rotation is clockwise, so a 90° rotation means that the entity will be facing downwards and not upwards, and vice versa for a 270° rotation